bin/completion-handlers.ts

```diff
@@ -0,0 +1,64 @@
+import { Completion } from '../src/index.js';
+
+export function setupCompletionForPackageManager(
+  packageManager: string,
+  completion: Completion
+) {
+  if (packageManager === 'pnpm') {
+    setupPnpmCompletions(completion);
+  } else if (packageManager === 'npm') {
+    setupNpmCompletions(completion);
+  } else if (packageManager === 'yarn') {
+    setupYarnCompletions(completion);
+  } else if (packageManager === 'bun') {
+    setupBunCompletions(completion);
+  }
+}
+
+export function setupPnpmCompletions(completion: Completion) {
+  completion.addCommand('add', 'Install a package', [], async () => []);
```

Comment:
for root command completions, you can do `exec('pnpm -h')` or something like that, and parse the output and that'd give you all the possible commands and options, so no need to write them by handle.

same with other package managers.

Reply:
still, i think we should parse this instead of doing things by hand.

![CleanShot 2025-05-05 at 11  17 09@2x](https://github.com/user-attachments/assets/8d4eb1bd-b9d3-4b15-8b16-a78e0f01bef1)

Reply:
the same should be done for npm and yarn